src/AtomicLockBundle/Cronjobs/TCMSCronjob_ClearAtomicLocks.class.php

```diff
@@ -9,8 +9,14 @@
  * file that was distributed with this source code.
  */
 
-class TCMSCronjob_ClearAtomicLocks extends TCMSCronJob
+class TCMSCronjob_ClearAtomicLocks extends TdbCmsCronjobs
 {
+
+    public function __construct()
```

Comment:
Empty constructors are not needed. (12 other occurrances here)

Reply:
all removed.